Sources/StreamChatSwiftUI/ChatThreadList/ChatThreadListScreen.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+/// Screen component of the thread list.
+/// It's the easiest way to integrate the SDK, but it provides the least customization options.
+/// Use the `ChatThreadListView` for more customizations.
+public struct ChatThreadListScreen: View {
```

Comment:
I think we said we will delete this? Since long term, we will not go with these screens anymore.

Reply:
Yeah I forgot to remove it 👍

Reply:
Done ✅